pages/index.vue

```diff
@@ -148,6 +162,30 @@ export default defineComponent({
      * @todo 2. Update the task in the dom.
      */
     updateTask(_index, _id) {},
+    updateTask(_index, _id) {
+      // console.log(this.todos[_index].title)
+      if (this.todos[_index].title === '') {
+        this.$toast.error('Task cannot be empty!!')
+      } else {
```

Comment:
A check could be added to ensure that the todo title is changed and not the same as the original one. This would prevent sending void calls to the backend.